### ISSUE TITLE
Unit Tests for ImmutableStatistics.java

### ISSUE THREAD

**Issue by YarinSh** [TechnionYP5779/team1#109]

**Comment by yoavz1997** [comment]
Consider adding cases in each one or both of the parameters to eq are null

**Comment by YarinSh** [comment]
yes it is already done, i just change the value of s1 and s2 all the time, I will change that, I changed that so it would be more understandable, but you can see that tide.java is covered 100%. @yoavz1997

**Comment by yoavz1997** [comment]
![image](https://user-images.githubusercontent.com/33288016/48568754-b75aee00-e908-11e8-9138-ec33834cd7e1.png)
I pulled a few times, maybe you forgot to push?

**Comment by YarinSh** [comment]
Yes the tideTest for some reason is not 100%, but tide.java is. I have no clue why it is so.
